module/VuFind/src/VuFind/Content/Covers/PluginManager.php

```diff
@@ -61,6 +61,7 @@ class PluginManager extends \VuFind\ServiceManager\AbstractPluginManager
         'orb' => Orb::class,
         'summon' => Summon::class,
         'syndetics' => Syndetics::class,
+        'bokinfo' => Bokinfo::class,
```

Comment:
I try to keep the configuration lists in alphabetical order; do you mind moving the factory and alias lines up?

Reply:
Do not mind at all.